### ISSUE TITLE
[Feature] List view of installed scripts

### ISSUE THREAD

**Issue by Eiion** [violentmonkey/violentmonkey#2174]
**Motivation**
Better overview of installed scripts

**Proposed Solution**
Rather than displaying installed scripts in two columns and multiple lines, give us the option to display installed scripts one below the other in a plain list without columns. Since each entry currently shows two lines, putting the info in one single lines will make it so that the total of installed scripts won't need any more screen space than it currently does but the overview of installed scripts will dramatically improve.

**Use Cases**
When managing installed scripts.

**Comment by tophf** [comment]
![image](https://github.com/user-attachments/assets/3a47ba5f-e6be-461f-bd8d-9331fb9a1254)

**Comment by Eiion** [comment]
Thanks for the pointer.

There's still an issue with the list view though - the screen space is used in an incredibly wasteful manner. Here's an exmaple:
![image](https://github.com/user-attachments/assets/69fc7570-6300-423d-8023-f77507b6587b)
With only 16 scripts installed I have to scroll on a 1440p screen resolution to see the last one. With all the wasted empty space between script name and creator/version/last update/size/delete info I don't see why this isn't setup in a way more efficient way.

Could you please make the list view more useful than it currently is? Maybe by e.g. moving the icons to edit/links to the center of the line as well as making the script icon much smaller to slim the lines down so the list view offers a more economic use of the screen and a quality of life update by not having to scroll through the list because the lines take a huge amount of screen space?

**Comment by tophf** [comment]
Your screenshot is not a table view that I suggested.

**Comment by Eiion** [comment]
The fact that you are mentioning table view leads me to believe that you did not understand the problem this issue pointed out nor what I said in my last reply. I wasn't asking for a table (i.e. mutliple column) but a list with slim lines.

To avoid any further misunderstandings: 
1. You didn't really suggest anything besides where to find settings to change the view, as you showed both options encircled and didn't write anything along it. So that's what I took your screenshot for, a "Here are the settings to get what you want". Therefore that's where I went to check and used to change from what I originally had to what I showed in my last reply with the screenshot.
Just for further clarification, here's what I originally had before changing to what I have shown in my last reply:
![image](https://github.com/user-attachments/assets/9ca59db0-7caa-41c0-81d8-03ed400706ed)
Keep in mind that this was with the settings as shown in your screenshot, right after installing Violentmonkey and importing the scripts - so I assume these are the default settings.)
3. I assume what you are referring to as "table view" is commonly called "grid view" - because it's a grid, not a table. What I refer to as "list view" is what I believe the setting in your screenshot refers to as "single column" because, well, one single column of information is a list. (If neither of that is what is meant by the words used in the options, a rewording might help communicate what is actually meant with the settings.)
Anyways, that just for clarification. I already have uninstalled Violentmonkey again and went back to using Tampermonkey as both views shown in my screenshots are impractical to me (hence the issue to request a feature and the request in my previous reply).

Again, this reply is just an attempt to clarify likely misunderstandings. Thanks for the previous replies.
